.circleci/config.yml

```diff
@@ -296,8 +296,8 @@ jobs:
       - restore_cache:
           keys:
               - tox-cache-django-{{ checksum "tox.ini" }}
-      - run: tox -e '{py27,py34,py35,py36}-django{18,19,110,111}-djangopylibmc06-djangoredis45-pylibmc-redis{210}-memcached' --result-json /tmp/django.1.results
-      - run: tox -e '{py27,py34,py35,py36}-django-autopatch{18,19,110,111}-djangopylibmc06-djangoredis45-pylibmc-redis{210}-memcached' --result-json /tmp/django.2.results
+      - run: tox -e '{py27,py34,py35,py36}-django{18,19,110,111}-djangopylibmc06-djangoredis45-pylibmc-redis{210}-memcached-restframework' --result-json /tmp/django.1.results
```

Comment:
I think we don't need all of this. Can you rollback this change and just create two new envs?
```
tox -e '{py27,py34,py35,py36}-django{18,19,110,111}-restframework{37,36,...}
tox -e '{py27,py34,py35,py36}-django-autopatch{18,19,110,111}-restframework{37,36,...}
```

To test rest framework we don't need memcache (make the required changes in the `settings.py` if needed, or create simply a new config file just for DRF).

Reply:
Also let's test the latest 3 or 4 versions of DRF, though having a best-effort support since 3.0 would be great.